src/main/scala/io/github/teonistor/draughts/Game.scala

```diff
@@ -13,7 +13,7 @@ class Game(val availableMovesRule: AvailableMovesRule,
   lazy val availableMoves: AvailableMoves = availableMovesRule.computeAvailableMoves(gameState, settings)
   lazy val isGameOver: Boolean = gameOverChecker.isGameOver(gameState, availableMoves)
 
-  def move(from: Position, to: Position): Validation[String,Game] = {
+  def move(from: Vector[Int], to: Vector[Int]): Validation[String,Game] = {
```

Comment:
I'm curious, why didn't you choose to make Position n-dimensional instead of having an explicit n-way vector? With a vector, it's always possible to end up with a move from n-dimensional space to, say, n-1

Reply:
I don't think there would be much benefit; given `n` can be any number (> 1), `Position` would pretty much become a vector without all the features.

At one point in the refactoring process `Position` was a type alias for `Vector[Int]` - perhaps I should've kept that...